pxtlib/gallery.ts

```diff
@@ -63,6 +64,15 @@
 
         return {};
     }
+    export function parseSimThemeJSON(md: string): Partial<pxt.PackageConfig> {
+        const pm = /```simtheme\s+((.|\s)+?)\s*```/i.exec(md);
```

Comment:
code ql is upset at this line, I'll fix it here for now but it is an existing regex (in this file, `parseTemplateProjectJSON`, `parseJResFromMarkdown`, `parseFeaturesFromMarkdown`, `parsePackagesFromMarkdown`) and in theory we should probably fix in all of those as well separately. Here's a build with the fix for this case => `/```simtheme\s+([\s\S]*?)```/i`, since we don't really care about trailing whitespace here as JSON.parse will just ignore it (though the worst case we should hit in this path would be the tutorial not loading, which there are already plenty of ways to make happen :) )

https://arcade.makecode.com/app/841da135762055afde2037ec09857edab2cc65ae-e8f3c30598#tutorial:44182-86552-65080-62109